Content.Shared/Xenoarchaeology/Artifact/XAT/XATExaminableTextSystem.cs

```diff
@@ -0,0 +1,27 @@
+using Content.Shared.Examine;
+using Content.Shared.Xenoarchaeology.Artifact.Components;
+using Content.Shared.Xenoarchaeology.Artifact.XAT.Components;
+
+namespace Content.Shared.Xenoarchaeology.Artifact.XAT;
+
+/// <remarks>
+/// This isn't an actual trigger but this framework is so fucking nice
+/// </remarks>
+public sealed class XATExaminableTextSystem : BaseXATSystem<XATExaminableTextComponent>
+{
```

Comment:
currently unused - i'm not sure why it should be used. Maybe to say stuff about next possible activations(then any1 can actvate complex stuff, this is bad)? or state(but its only 10 sec - who will spend time for that)? maybe remove for now?